src/main/java/com/hedvig/notificationService/customerio/state/InMemoryCustomerIOStateRepository.kt

```
@@ -16,11 +16,11 @@ class InMemoryCustomerIOStateRepository(var data: Map<String, CustomerioState> =
     override fun shouldUpdate(byTime: Instant): Collection<CustomerioState> {
         return data.values.filter {
             (it.underwriterFirstSignAttributesUpdate != null &&
-                it.underwriterFirstSignAttributesUpdate <= byTime &&
+                it.underwriterFirstSignAttributesUpdate!! <= byTime &&
```

Do we need to throw an exception here? or just return null?

---

This is fine, this is the InMemoryRepository that we use just for testing.